lib/taskana-cdi-example/src/main/java/pro/taskana/example/ExampleBootstrap.java

```diff
@@ -1,57 +1,63 @@
 package pro.taskana.example;
 
-import javax.annotation.PostConstruct;
-import javax.ejb.EJB;
-import javax.enterprise.context.ApplicationScoped;
-import javax.enterprise.context.Initialized;
-import javax.enterprise.event.Observes;
-import pro.taskana.classification.api.exceptions.ClassificationNotFoundException;
-import pro.taskana.common.api.exceptions.InvalidArgumentException;
-import pro.taskana.task.api.exceptions.AttachmentPersistenceException;
-import pro.taskana.task.api.exceptions.InvalidOwnerException;
-import pro.taskana.task.api.exceptions.InvalidTaskStateException;
-import pro.taskana.task.api.exceptions.ObjectReferencePersistenceException;
-import pro.taskana.task.api.exceptions.TaskAlreadyExistException;
-import pro.taskana.task.api.exceptions.TaskNotFoundException;
+import jakarta.ejb.EJB;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.event.Startup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pro.taskana.classification.api.models.Classification;
 import pro.taskana.task.api.models.Task;
 import pro.taskana.task.internal.models.ObjectReferenceImpl;
-import pro.taskana.workbasket.api.exceptions.NotAuthorizedOnWorkbasketException;
-import pro.taskana.workbasket.api.exceptions.WorkbasketNotFoundException;
+import pro.taskana.workbasket.api.WorkbasketType;
+import pro.taskana.workbasket.api.models.Workbasket;
 
 /** Example Bootstrap Application. */
 @ApplicationScoped
 public class ExampleBootstrap {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ExampleBootstrap.class);
+
+  private static final String CDIDOMAIN = "CDIDOMAIN";
+  private static final String CLASSIFICATION_TYPE = "T1";
+
   @EJB private TaskanaEjb taskanaEjb;
 
-  @PostConstruct
-  public void init(@Observes @Initialized(ApplicationScoped.class) Object init)
-      throws TaskNotFoundException,
-          WorkbasketNotFoundException,
-          ClassificationNotFoundException,
-          InvalidOwnerException,
-          TaskAlreadyExistException,
-          InvalidArgumentException,
-          AttachmentPersistenceException,
-          ObjectReferencePersistenceException,
-          NotAuthorizedOnWorkbasketException,
-          InvalidTaskStateException {
-    System.out.println("---------------------------> Start App");
+  /**
+   * The parameter `@Observes Startup` makes sure that the dependency injection framework calls this
+   * method on system startup. And to do that, it needs to call `@PostConstruct start()` first.
+   *
+   * @param startup just the startup event
+   * @throws Exception in case of task creation fails
+   */
+  public void init(@Observes Startup startup) throws Exception {
+    LOGGER.info("---------------------------> Start App -- {}", startup);
+    Workbasket workbasket = taskanaEjb.getWorkbasketService().newWorkbasket("KEY", CDIDOMAIN);
```

Comment:
Why do add Workbasket and Classification here? It looks better this way to me :)

Reply:
Is this a compliment? 
if yes -> :heart: 
if no -> :disappointed: 

I extended the testcase to verify the deployment and the execution of the PostConstruct method from BootstrapExample 

To create a Task correctly you need to assign it with a Workbasket and a classification. There is no default so far I know. The BootstrapExample did not work at all before. The test just triggered the deployment but did not verify the execution of the postconstruct.